<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Zgame</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QTextEdit" name="outputTextEdit">
                        <property name="readOnly">
                            <bool>true</bool>
                        </property>
                    </widget>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="buttonLayout">
                        <item>
                            <widget class="QPushButton" name="forwardButton">
                                <property name="text">
                                    <string>Forward</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="backButton">
                                <property name="text">
                                    <string>Back</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="leftButton">
                                <property name="text">
                                    <string>Left</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="rightButton">
                                <property name="text">
                                    <string>Right</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <layout class="QHBoxLayout" name="actionButtonLayout">
                        <item>
                            <widget class="QPushButton" name="searchButton">
                                <property name="text">
                                    <string>Search</string>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QPushButton" name="takeButton">
                                <property name="text">
                                    <string>Take</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </item>
                <item>
                    <widget class="QLineEdit" name="commandLineEdit"/>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
